docs/modules/ROOT/pages/patterns.adoc

```diff
@@ -187,24 +177,24 @@ This translates to:
 ----
 
 Note how the initial node in the pattern (`this0`) is the same as the one referenced in the last element. 
-This matches actors that also directed the same movie.
+This matches actors who also directed the same movie.
 
 === Length
 
-The length (or hops) of a relationship can be defined with the `withLength` method.
+The length (or hops) of a relationship can be defined with the `length` property.
 
 ==== Exact length
 
 The exact length can be defined by passing a number:
 
 [source, javascript]
 ----
-const pattern = new Cypher.Pattern(person).related(actedIn).withLength(3).to(movie);
+const pattern = new Cypher.Pattern({}).related(actedIn, { type: "ACTED_IN", length: 3 }).to();
```

Comment:
is this not even available in the shorter form `new Cypher.Pattern()` ?

Reply:
Not until we deprecate the old version to maintain compatibility :disappointed: